pyas2/management/commands/sendas2message.py

```diff
@@ -66,13 +67,21 @@ def handle(self, *args, **options):
                 content_type=partner.content_type,
                 disposition_notification_to=org.email_address or "no-reply@pyas2.com",
             )
+
         message, _ = Message.objects.create_from_as2message(
             as2message=as2message,
             payload=payload,
             filename=original_filename,
             direction="OUT",
             status="P",
         )
+
+        # Check if we're inside an atomic block, if not, commit immediately to store the message
+        if not transaction.get_connection().in_atomic_block:
```

Comment:
what is the reason for this change?

Reply:
I had this in my branch and was a left over from here https://github.com/abhishek-ram/django-pyas2/issues/45. Issue is, the message is not stored before sending, and if partner sends an MDN back before the message is commited to DB, then the MDN fails and triggers a "no message found" error. This should prevent this.

Reply:
The reason for this is, because tests fun in atomic transaction and therefore this will fail. The behavior of the django test is different than when running the command "normally". transaction.commit() would fail if running inside an atomic transaction.

Reply:
We should not be writing code to handle for tests, this needs to be handled differently. Can be picked later

Reply:
We don't know how anyone would be using this command - someone might wrap this command in an atomic transaction which would then fail this too probably. So I don't see this as "for tests"... 
Originally I was going to use a config flag if we are in test mode or not, but then we even have more "only test specific" code.

Let me know if you want me to simple remove this from this PR, then I will and you can decide at a later point how you want this.

Reply:
lets move  it out of the PR

Reply:
done